dill/_dill.py

```diff
@@ -341,11 +338,6 @@ class Pickler(StockPickler):
     :meta hide-value:
     """
     from .settings import settings
-    # Flags set by dump_module() is dill.session:
-    _refimported = False
-    _refonfail = False
-    _session = False
```

Comment:
seems `_session` shouldn't have been deleted -- it's still used in `_locate_function` and `_save_module_dict`.